### PR TITLE
(GH-128) Fixes broken link to portable package

### PR DESCRIPTION
**Pull request by trncb**
This fixes issue #128 to change the URL to be inline with the Markdown file name, and updated the correct anchor name.

### CODE REVIEW

File: GettingStarted.md [hunk 1]
```diff
@@ -146,7 +146,7 @@ Software and Package are not terms used interchangeably in the Chocolatey commun
 * **Software** - Software refers to the actual runtime software that a package represents. This can be installed to the system through native installers, or come from zip/archive files or just dropping the runtime software right into the package.
 * **Native Installer** - Actual installers that install software into Programs and Features, "natively" on a Windows machine. This is like MSI, InnoSetup (exe), NSIS (exe), InstallShield (exe/msi), etc. Windows has over 20 different installer formats.
 * **Install Package** - packages that wrap native installers
-* **Portable Package** - packages that use zip or just contain the runtime software. Usually these packages do not require administrative privileges to install or run. See [[|https://github.com/chocolatey/choco/wiki/ChocolateyFAQs#portable-application-something-that-doesnt-require-a-system-install-to-use]]
+* **Portable Package** - packages that use zip or just contain the runtime software. Usually these packages do not require administrative privileges to install or run. See [[Portable application|ChocolateyFAQs#portable-application--something-that-doesnt-require-a-system-install-to-use]]
```
Comment (gep13) [inline GettingStarted.md]:
```suggestion
* **Portable Package** - packages that use zip or just contain the runtime software. Usually these packages do not require administrative privileges to install or run. See [[Portable application|faq#portable-application--something-that-doesnt-require-a-system-install-to-use]]
```

I "think" that this is the correct URL to point people at.

Comment (trncb) [reply GettingStarted.md]: I don’t think so @gep13.

I followed the format used (mostly) throughout the files in the repo. ie. ‘[[LinkDisplay|NameOfMarkdownFile#AnchorName]]

**Comment by gep13** [comment]
@trncb it could be that I am misremembering how the HTML is rendered from the markdown.  Let's give it a try and see how it works out 👍

**Comment by gep13** [comment]
@trncb I have just tested this, and it works as expected.  Thank you for the submission, and for pointing out my mistake!

**Pull request merged**